Hi @t-b. All travis jobs are failed due to the following issue:
```
xports.def : warning LNK4022: cannot find unique match for symbol '__' [C:\projects\cppTango\tango.vcxproj]
exports.def : warning LNK4002: __@@_PchSym_@00@UkilqvxghUxkkgzmtlUxkkzkrUhvieviUhvieviPlyqvxghOwriUivovzhvUkxsPhvieviPlyqvxghOlyq@4B2008FD98C1DD4 defined in C:\projects\cppTango\cppapi\server\server_objects.dir\Release\pch_server_objects.obj [C:\projects\cppTango\tango.vcxproj]
exports.def : warning LNK4002: __@@_PchSym_@00@UkilqvxghUxkkgzmtlUxkkzkrUxorvmgUxorvmgPlyqvxghOwriUivovzhvUkxsPxorvmgPlyqvxghOlyq@4B2008FD98C1DD4 defined in C:\projects\cppTango\cppapi\client\client_objects.dir\Release\pch_client_objects.obj [C:\projects\cppTango\tango.vcxproj]
exports.def : error LNK2001: unresolved external symbol __ [C:\projects\cppTango\tango.vcxproj]
C:/projects/cppTango/Release/tango.lib : fatal error LNK1120: 1 unresolved externals [C:\projects\cppTango\tango.vcxproj]
Command exited with code 1
```
I have no idea why we see this error. We have special handling in PCH builds to prevent this issue:
https://github.com/tango-controls/cppTango/blob/tango-9-lts/cppapi/client/CMakeLists.txt#L66

I don't have permissions to restart the job. Can you do this? Other PRs are building fine on appveyor.

--- a/cppapi/client/zmqeventconsumer.cpp
+++ b/cppapi/client/zmqeventconsumer.cpp
@@ -1261,21 +1261,29 @@ void ZmqEventConsumer::connect_event_channel(std::string &channel_name,TANGO_UNU
 // If the server has returned several possible ZMQ endpoints (because several NIC boards on server host), check which
 // one is correct
 //
+    size_t nb_endpoints = ev_svr_data->svalue.length();
 
-    size_t nb_endpoint = ev_svr_data->svalue.length();
-    nb_endpoint = nb_endpoint >> 1;
+    Tango::DevLong server_tango_lib_version = ev_svr_data->lvalue[0];
+    if(server_tango_lib_version >= 930)
+    {
+        // ZmqEventSubscriptionChange returns the ZMQ message filters used for the attribute
+        // and for the heartbeat events at the end of the string array part of the returned DevVarLongStringArray
+        nb_endpoints -= 2;
+    }
+
+    nb_endpoints = nb_endpoints >> 1;
     size_t valid_endpoint = 0;
 
-    if (nb_endpoint != 1)
+    if (nb_endpoints != 1)
     {
-        for (valid_endpoint = 0;valid_endpoint < nb_endpoint;valid_endpoint++)
+        for (valid_endpoint = 0; valid_endpoint < nb_endpoints; valid_endpoint++)
         {
             std::string endpoint(ev_svr_data->svalue[valid_endpoint << 1]);
             if (check_zmq_endpoint(endpoint) == true)
                 break;
         }
 
-        if (valid_endpoint == nb_endpoint)
+        if (valid_endpoint == nb_endpoints)
         {
             std::stringstream o;
 

